Quartz/ThirdParty/CMakeLists.txt

```diff
@@ -13,12 +13,9 @@ endif()
 
 set(SDL_SHARED_ENABLED_BY_DEFAULT OFF)
 
-add_subdirectory(glad)
 add_subdirectory(SDL2)
-add_subdirectory(imgui)
```

Comment:
because of this we'll have to start again the launcher stuff

Reply:
TBH I am still thinking we want to do the launcher 100% separate. Ideally when Quartz updates the launcher shouldn't.